fearless/Modules/Staking/StakingBalance/ViewModel/StakingBalanceViewModelFactory.swift

```diff
@@ -179,13 +179,14 @@ struct StakingBalanceViewModelFactory: StakingBalanceViewModelFactoryProtocol {
     private func timeLeftAttributedString(
         activeEra: EraIndex,
         unbondingEra: EraIndex,
-        eraCompletionTime: TimeInterval?,
+        eraCountdown: EraCountdown?,
         locale: Locale
     ) -> NSAttributedString {
         let eraDistance = unbondingEra - activeEra
         let daysLeft = Int(eraDistance) / chain.erasPerDay
```

Comment:
Probably now we don't need `erasPerDay` because we can pass target era to `eraCompletionTime`.